hugegraph-core/src/main/java/com/baidu/hugegraph/backend/tx/SchemaTransaction.java

```diff
@@ -444,6 +447,15 @@ public void checkIdIfRestoringMode(HugeType type, Id id) {
         }
     }
 
+    private static void setCreateTime(SchemaElement schema) {
+        if (schema instanceof IndexLabel) {
```

Comment:
why not create for IndexLabel

Reply:
IndexLabel doesn't support userdata

Reply:
prefer to let IndexLabel support userdata